webapp/channels/src/components/global_header/left_controls/product_menu/product_menu_list/product_menu_list.tsx

```diff
@@ -128,25 +134,15 @@ const ProductMenuList = (props: Props): JSX.Element | null => {
                                 )}
                             </>
                         )}
-                        icon={
-                            <Icon
-                                size={16}
-                                glyph={'application-cog'}
-                            />
-                        }
+                        icon={<ApplicationCogIcon size={18}/>}
```

Comment:
Is it okay to change the size from `16` to `18`?

Reply:
Yeah, I checked the sizes were not same in HTML

Reply:
I had to bump size in for all Icons